### PR TITLE
Add support for binary on/off led types

### PR DESCRIPTION
**Pull request by spiiroin**

### CODE REVIEW

**Comment by sledges** [comment]
LGTM

File: hybris.c [hunk 1]
```diff
@@ -1487,6 +1557,84 @@ led_control_htcvision_probe(led_control_t *self)
...
+led_control_binary_close_cb(void *data)
+{
+  led_state_binary_t *state = data;
+  led_state_binary_close(state + 0);
+}
+
+static bool
+led_control_binary_probe(led_control_t *self)
+{
+  /** Sysfs control paths for RGB leds */
```
Comment (saukko) [inline hybris.c]: RGB? :)

File: hybris.c [hunk 2]
```diff
@@ -1487,6 +1557,84 @@ led_control_htcvision_probe(led_control_t *self)
...
+  static const led_paths_binary_t paths[][1] =
+  {
+    // binary
+    {
+      {
+        .val = "/sys/class/leds/button-backlight/brightness",
```
Comment (saukko) [inline hybris.c]: We might want this to be configurable at some point, but probably not needed now.

File: hybris.c (hunk 1 above)
Comment (spiiroin) [inline hybris.c]: I'll fix that before tagging

**Pull request merged**